pkg/identity/service.go

```diff
@@ -222,7 +222,7 @@ func (svc *Service) Certify(ctx context.Context, req *pb.CertifyRequest) (*pb.Ce
 	if reqIdentity != tokIdentity {
 		msg := fmt.Sprintf("requested identity did not match provided token: requested=%s; found=%s",
 			reqIdentity, tokIdentity)
-		log.Debug(msg)
+		log.Infof(msg)
```

Comment:
We could probably inline it all with `log.Infof("requested identity did not match provided token: requested=%s; found=%s", reqIdentity, tokIdentity)` and get rid of msg.

Reply:
we use the `msg` in the status

Reply:
Then you wanna use log.Info instead 😉